app.rb

```diff
@@ -42,7 +42,7 @@
 
       unless issue.rejected?
         issue.set_version(current_version['id']) if issue.version.nil? && current_version
-        issue.set_pull_request(pull_request.raw_data['html_url']) if issue.pull_request.nil? || issue.pull_request.empty?
+        issue.add_pull_request(pull_request.raw_data['html_url']) if issue.pull_request.nil? || issue.pull_request.empty?
```

Comment:
That actually points out something I was missing which is to remove the nil/empty check since we want to add to the set of pull requests in all cases.